the-blue-alliance-ios/View Elements/Awards/AwardTableViewCell.swift

```diff
@@ -40,25 +55,28 @@ class AwardTableViewCell: UITableViewCell, Reusable {
         removeAwards()
 
         for (index, recipient) in viewModel.recipients.enumerated() {
-            let stackView = UIStackView()
-            stackView.axis = .vertical
-            stackView.tag = index
-            stackView.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(recipientTapped(gesture:))))
-            for text in recipient.awardText {
-                let label = boldLabelWithText(text)
-                stackView.addArrangedSubview(label)
+            var viewToAdd = UIView()
+            if let teamNumber = recipient.teamNumber, let awardee = recipient.awardee {
+                let button = AwardTeamButton(header: teamNumber, subheader: recipient.teamName ?? "", widthRange: (UIScreen.main.bounds.width * 0.3)...(UIScreen.main.bounds.width * 0.4))
```

Comment:
Oh no... we shouldn't be doing this. We should be able to use Auto Layout to set sizes like this.